src/gwt/src/org/rstudio/studio/client/workbench/views/console/model/ConsoleServerOperations.java

```diff
@@ -36,6 +36,8 @@ void consoleInput(String consoleInput,
                      int flags,
                      ServerRequestCallback<Void> requestCallback);
    
+   void consoleFollowHyperlink(String url, String link_params, ServerRequestCallback<Void> requestCallback);
```

Comment:
Is this the right class for this ?

Reply:
I think this is fine, but if you felt strongly then a separate `VirtualConsoleServerOperations` class might be a better fit.

Reply:
At least as a learning exercise, I'll try to make it a separate class/interface.